scripts/install-fabric.sh

```diff
@@ -268,29 +267,41 @@ CA_BINARY_FILE=hyperledger-fabric-ca-${ARCH}-${CA_VERSION}.tar.gz
 
 # if nothing has been specified, assume everything
 if [[ ${_arg_comp[@]} =~ ^$ ]]; then
-    echo "Getting all samples, binaries, and docker images"
+    echo "No options selected: Getting all samples, binaries, and docker images"
+    echo "Abort now if not the intention"
+    sleep 3 # just to give a chance to abort if this wasn't intended
     _arg_comp=('samples','binary','docker')
 fi
 
 # Process samples first then the binaries. So if the fabric-samples dir is present
 # the binaries will go there
-if [[ "${_arg_comp[@]}" =~ s(amples)? ]]; then
+if [[ "${_arg_comp[@]}" =~ ^s(amples)? ]]; then
         echo
         echo "Clone hyperledger/fabric-samples repo"
         echo
         cloneSamplesRepo
 fi
 
-if [[ "${_arg_comp[@]}" =~ b(inary)? ]]; then
+if [[ "${_arg_comp[@]}" =~ ^b(inary)? ]]; then
         echo
         echo "Pull Hyperledger Fabric binaries"
         echo
         pullBinaries
 fi
 
-if [[ ${_arg_comp[@]} =~ d(ocker)? ]]; then
+if [[ "${_arg_comp[@]}" =~ ^p(odman)? ]]; then
+        echo
+        echo "Pull Hyperledger Fabric podman images"
```

Comment:
They are not "podman images" or "docker images". They are images pulled using either docker command or podman command, right?

Reply:
that's correct

Reply:
Yes that's correct.... so it would be more accurate to say download the container images or not.. and what tool to choose.

Docker has  like 'hover' become a generic term for container/image... but is wrong...